src/account/saga.ts

```diff
@@ -79,7 +78,6 @@ export function* initializeAccountSaga() {
     ValoraAnalytics.track(OnboardingEvents.initialize_account_start)
     yield call(getOrCreateAccount)
     yield call(generateSignedMessage)
-    yield put(refreshAllBalances())
```

Comment:
i don't think this is needed, since the wallet home screen already dispatches this action

Reply:
I think we should leave this if we're not sure of all consequences with the different flows.